terracumber_config/tf_files/tfvars/PR-testing-uyuni.tfvars

```diff
@@ -2,7 +2,7 @@
 
 IMAGE                  = "opensuse155-ci-pro"
 SERVER_IMAGE           = "leapmicro55o"
-PROXY_IMAGE            = "opensuse155o"
+PROXY_IMAGE            = "leapmicro55o"
 IMAGES                 = ["rocky9o", "opensuse155o", "opensuse155-ci-pro", "ubuntu2204o", "sles15sp4o", "leapmicro55o"]
```

Comment:
Do we use "opensuse155o" somewhere?
Otherwise we will be downloading it for nothing

Reply:
good point. Let me create a separate PR